runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx

```diff
@@ -54,7 +54,7 @@ const Navbar: FC<NavbarProps> = ({
                         {translate('Landing.Header.Nav.Option.RPA')}
                     </Link>
                 </li>
-                <li className={styles.listItem}>
+                {/* <li className={styles.listItem}>
```

Comment:
almost never leave commented code

Reply:
this is acceptable, we want to hide part of the LP, thus header needs adjustment too